src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static class Task {
+        protected String description;
+        protected boolean isDone;
+        final static String tickSymbol =  "\u2713";
+        final static String crossSymbol = "\u2718";
+        public Task(String description) {
+            this.description = description;
+            this.isDone = false;
+        }
+        public String getStatusIcon() {
+            return (isDone ? tickSymbol : crossSymbol); //return tick or X symbols
+        }
+        public void markedAsDone(){
+            this.isDone = true;
+            printMarkMessage(this.description);
+        }
+        public void printMarkMessage(String description) {
+            printLine();
+            System.out.println("     Nice! I've marked this task as done:");
+            System.out.println("       ["+tickSymbol+"] "+description);//print tick and the task marked
+            printLine();
+        }
+        @Override
+        public String toString (){
+            return "["+this.getStatusIcon()+ "] "+description;
+        }
+    }
+    public static class Todo extends Task{
+
+        public Todo(String description) {
+            super(description);
+        }
+        @Override
+        public String toString(){
+            return "[T]" + super.toString();
+        }
+
+    }
+    public static class Deadline extends Task{
+        protected String by;
+        public Deadline(String description) {
+            super(description);
+        }
+        public void setBy(String by ){
+            this.by = by;
+        }
+        @Override
+        public String toString() {
+            return "[D]" + super.toString() + " (by: " + this.by + ")";
+        }
+    }
+    public static class Event extends Task{
+        protected String at;
+        public Event(String description) {
+            super(description);
+        }
+        public void setAt(String at ){
+            this.at = at;
+        }
+        @Override
+        public String toString(){
+            return "[E]"+super.toString()+" (at: "+this.at+")";
+        }
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        String command;
+        Task [] tasks = new Task[100];
+        int tasksTotal = 0;
+        Task taskToAdd = null;
+        Task taskToMark;
+        Scanner in = new Scanner(System.in);
+        command = in.nextLine();
+        while(!command.equals("bye")){
+            if(command.equals("list")) printList(tasks, tasksTotal);
```

Comment:
I think it will be better to include curly brackets and put the statement in a new line even though there is only one statement, what do you think?